tests/specs/lemmas.k

```diff
@@ -269,6 +278,9 @@ module LEMMAS-HASKELL [symbolic, kore]
     rule maxUInt160 &Int N => N
       requires #rangeUInt(160, N) [simplification]
 
+    rule N modInt pow160 => N
+      requires #rangeUInt(160, N) [simplification]
+
```

Comment:
Any reason for this to be in `LEMMAS-HASKELL` and not just `LEMMAS`?

Reply:
closer to its brothers above : D